JFrogVSExtension/Utils/Util.cs

```diff
@@ -185,6 +193,32 @@ public static Component ParseDependencies(Dependency dep, Dictionary<string, Art
             return comp;
         }
 
+        // Make sure that all of the component's dependencies are added to the DataService.
+        // This scenario might happen as a package may appear in the dependencies-tree numerous times with different dependencies.
+        // This is due to a possible situation of multiple subprojects under the same solution,
+        // where the same package appears in several projects but its dependencies varies between the packages.config files of the subprojects.
+        private static void updateDataServiceWithMissingDependencies(Component component, DataService dataService)
+        {
+            Component componentInDataService = dataService.getComponent(component.Key);
+            if (component.Dependencies == null || componentInDataService == null)
+            {
+                return;
+            }
+
+            if (componentInDataService.Dependencies == null)
+            {
+                componentInDataService.Dependencies = new List<string>();
+            }
+
+            foreach (string dependencyString in component.Dependencies)
+            {
+                if (componentInDataService.Dependencies != null && !componentInDataService.Dependencies.Contains(dependencyString))
```

Comment:
It loos like this check - ```componentInDataService.Dependencies != null``` is not needed, because we already took care of it in line 208.

Reply:
Removed the redundant null check.